File: Solutions/Python/pattern006.py
# Pattern Programs n.006 [Python]
# Github Repository: https://github.com/mxsena/pattern-programs
n = 5
k = 1

for x in range(1, n + 1):
    for y in range(1, n + 1):
        print("{:2d} ".format(k), end=" ")
        k += 1
    print()

File: Solutions/C/pattern006.c
// Pattern Programs n.006 [C]
// Github Repository: https://github.com/mxsena/pattern-programs
#include <stdio.h>

int main(void)
{
    int rows = 5, columns = 5;
    int num = 1;

    for (int i = 1; i <= rows; i++)
    {
        for (int j = 1; j <= columns; j++)
        {
            printf("%3d ", num);
            num++;
        }
        printf("\n");
    }
    return 0;
}